mantidproject/mslice
# Add 2D Workspaces

**Not to be merged until after #220**

Loaded unprojected workspaces can now be combined using the 'Add' button. This uses the `MergeRuns` algorithm. If only one workspace is selected, a dialog appears asking for another. Otherwise, all selected workspaces are passed to `MergeRuns`.

**To test:**
- Load one or more workspaces and select in the workspace manager
- Click 'Add'
- Check new workspace with _sum appended appears.

Fixes #223
